setup.py

```diff
@@ -53,7 +53,7 @@
 base_dir = os.path.dirname(os.path.abspath(__file__))
 third_party_path = os.path.join(base_dir, 'third_party')
 
-_libtpu_version = '0.1.dev20220816'
+_libtpu_version = '0.1.dev20220928'
```

Comment:
nit: to keep tidy control over changes, please add something like "libtpu pin update" to the title. I added the `libtpu-update` tag to this PR.

Reply:
Updated the PR title. Will include that in the commit message too.